flink/v1.20/flink/src/test/java/org/apache/iceberg/flink/data/TestRowProjection.java

```diff
@@ -369,19 +346,7 @@ public void testMapProjection() throws IOException {
     assertThat(projected.getMap(0)).isEqualTo(properties);
   }
 
-  private Map<String, ?> toStringMap(Map<?, ?> map) {
```

Comment:
removing this seems to be missing from the same file in Flink 1.19

Reply:
the ide didn't flag it as unused there... turns out it was too lazy.

thanks for catching!